ndk/Cargo.toml

```diff
@@ -47,6 +47,10 @@ package = "ndk-sys"
 path = "../ndk-sys"
 version = "0.5.0-beta.0"
 
+[dev-dependencies]
+# Only for use in documentation and doc-tests
+libc = "0.2"
```

Comment:
Before merging this - if folks find this a good idea at all - I'd like to go over existing documentation and replace their mentions of e.g. `mmap` and other flags with `libc` intradoc links.

Reply:
Appears this is not true at all. `dev-dependencies` applies to (doc!) tests, but not to documentation itself.

I can add it behind the `docsrs` cfg then... :(

Reply:
Note too that many of these constants - except the `PROT_` ones, are (accidentally?) included in the `ndk-sys` ffi bindings...

But better to reference public/obvious ones from `libc` unless there's a "kernel header" target?